app/services/word_watcher.rb

```diff
@@ -7,6 +7,19 @@ def initialize(raw)
     @raw = raw
   end
 
+  def self.disable_cache
+    @disabled = true
```

Comment:
The variable name is a bit confusing. What is being disabled? The `WordWatcher` or the cache? 🤔 

Maybe we should rename it to `@cache_disabled` instead?